sdk/analytics/src/test/java/com/klaviyo/analytics/model/KeywordsTest.kt

```diff
@@ -60,10 +67,16 @@ class KeywordsTest {
         assertEquals("\$failed_payment", EventType.FAILED_PAYMENT.name)
 
         val expectedCustomKey = Math.random().toString() + "_key"
+        assertEquals(expectedCustomKey, EventMetric.CUSTOM(expectedCustomKey).name)
         assertEquals(expectedCustomKey, EventType.CUSTOM(expectedCustomKey).name)
 
         // Test the equals operator works properly on custom keys
-        val custom = EventType.CUSTOM(expectedCustomKey)
+        var custom: EventMetric = EventMetric.CUSTOM(expectedCustomKey)
+        assert(custom == EventMetric.CUSTOM(expectedCustomKey))
+        assert(custom != EventMetric.CUSTOM(expectedCustomKey + "1"))
+
+        // Test the equals operator works properly on custom keys
+        custom = EventType.CUSTOM(expectedCustomKey)
```

Comment:
Other tests seem to rip out `EventType` entirely. Is there a reason this test in particular is testing the deprecated `EventType` as well? I feel like if we want to keep tests for some deprecated logic around it may be better doing it in a standalone test separate from the test checking `EventMetrics`

Reply:
Total nit since this test doesnt really do much in the first place

Reply:
The v2 pull request does rip out the `EventType` tests bc that PR is for the breaking changes where I'm deleting `EventType`.

Reply:
Just referring to the other tests like those in `KlaviyoTest` and `KlaviyoApiClientTest` where the use of `EventType` was replaced. But ya, they are literally just changing which object is called - its otherwise identical logic - so it really doesn't matter.

But ya its just a change in which object is called so thinking about it more I really dont think it matters

Reply:
Ooh I see what you mean, yeah that's actually a good point I'm probably missing test coverage now.

Reply:
Gonna fix that juuust to be safe.

Reply:
Added back a test against EventType in `KlaviyoTest` so coverage is 100% on event model initializers. Don't need to test both paths in the networking test stubs